app/config/collections.php

```diff
@@ -800,7 +800,7 @@
                 'size' => Database::LENGTH_KEY,
                 'signed' => true,
                 'required' => false,
-                'default' => null,
+                'default' => 0,
```

Comment:
This default should stay `null` 🙂

Reply:
Cool

Reply:
Oh, the default of `projectId` needs to be null, the default of `expire` should be `0` 👍🏻